Rakefile

```diff
@@ -6,11 +6,3 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
-
-if Rails.env.production?
-  # https://github.com/heroku/heroku-buildpack-ruby/pull/892#issuecomment-548897899
-  assets_precompile_task = Rake.application.tasks.find { |task| task.name == 'assets:precompile' }
-  if assets_precompile_task.present?
-    assets_precompile_task.prerequisites.delete('yarn:install')
-  end
-end
```

Comment:
The code being deleted here wasn't doing anything anymore. `yarn:install` was not anymore a prerequisite of `assets:precompile`. Rather, `yarn install` was being run because `vite:install_dependencies` is added to the `assets:precompile` rake task (which, as a side note, I think is now provided by `sprockets` rather than by `rails` itself).